BundleTransport/app/src/main/java/net/discdd/bundletransport/TransportWifiDirectService.java

```diff
@@ -1,5 +1,5 @@
 package net.discdd.bundletransport;
-
+import android.content.Context;
```

Comment:
you added this import, but i don't see it getting used. are you using the "organize imports" tool?